tests/test_adapters.py

```diff
@@ -10,19 +11,34 @@
     BatchedGenerationRequest,
 )
 
+try:
+    from vllm.entrypoints.openai.protocol import LoadLoRAAdapterRequest
+except ImportError:
+    from vllm.entrypoints.openai.protocol import (
+        LoadLoraAdapterRequest as LoadLoRAAdapterRequest,
+    )
+
 FIXTURES_DIR = Path(__file__).parent / "fixtures"
 
 
+@pytest.fixture
+def vllm_model_handler() -> AsyncMock:
+    mock_handler = AsyncMock()
+    return mock_handler
```

Comment:
I wonder if for this test it would be easier to write a class like this:
```
class FakeModelsHandler:

    def __init__(self):
        self.lora_requests = []

    async def load_lora_adapter(self, request):
        if request.lora_name.starts_with("poison"):
            raise ValueError("oops!")
        self.lora_requests.append( #make a LoRARequest from the LoadLoRAAdapterRequest)
```

Seems simpler than dealing with the mocks since we're only using this as a simple cache